packages/grafana-data/src/utils/tests/mockStandardProperties.ts

```diff
@@ -67,6 +67,23 @@ export const mockStandardProperties = () => {
     shouldApply: () => true,
   };
 
+  const localMinMax = {
+    id: 'localMinMax',
+    path: 'localMinMax',
+    name: 'localminmax',
```

Comment:
I somehow don't like the `local...` prefix to this name. What about `fieldMinMax` instead? A little bit more descriptive. Thinking about current `min`/`max` it's not clear that it's dataset's min/max. Maybe let's avoid these problem when introducing the new property :)

Reply:
That is a good point. Will fix that.